vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +95,50 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LanguageDescriptionObject title, ExhibitionAboutPage exhibitionAboutPage) {
+        if(title!=null) {
+
+            ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findByLabel(title.getExhibitionLanguage().getLabel());
```

Comment:
why `findByLabel`? why not `findById`?

Reply:
Hi @jdamerow , when I tried sending the exhibition language id from ui, I got a rejected value error. Need your help to resolve it .

16 Feb 2023 16:10:55,769 DEBUG: org.springframework.security.web.header.writers.HstsHeaderWriter - Not injecting HSTS header since it did not match the requestMatcher org.springframework.security.web.header.writers.HstsHeaderWriter$SecureRequestMatcher@2629343c
16 Feb 2023 16:10:55,770 WARN : org.springframework.web.servlet.mvc.support.DefaultHandlerExceptionResolver - Resolved exception caused by handler execution: org.springframework.validation.BindException: org.springframework.validation.BeanPropertyBindingResult: 4 errors
Field error in object 'aboutPageForm' on field 'titles[0].exhibitionLanguage.id': rejected value [LANG000000014]; codes [methodInvocation.aboutPageForm.titles[0].exhibitionLanguage.id,methodInvocation.aboutPageForm.titles.exhibitionLanguage.id,methodInvocation.titles[0].exhibitionLanguage.id,methodInvocation.titles.exhibitionLanguage.id,methodInvocation.id,methodInvocation.java.lang.String,methodInvocation]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [aboutPageForm.titles[0].exhibitionLanguage.id,titles[0].exhibitionLanguage.id]; arguments []; default message [titles[0].exhibitionLanguage.id]]; default message [Property 'titles[0].exhibitionLanguage.id' threw exception; nested exception is java.lang.StackOverflowError]
Field error in object 'aboutPageForm' on field 'titles[1].exhibitionLanguage.id': rejected value [LANG000000017]; codes [methodInvocation.aboutPageForm.titles[1].exhibitionLanguage.id,methodInvocation.aboutPageForm.titles.exhibitionLanguage.id,methodInvocation.titles[1].exhibitionLanguage.id,methodInvocation.titles.exhibitionLanguage.id,methodInvocation.id,methodInvocation.java.lang.String,methodInvocation]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes